docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_without_the_cli.rst

```diff
@@ -0,0 +1,55 @@
+.. _how_to_guides__creating_and_editing_expectations__how_to_create_a_new_expectation_suite_without_the_cli:
+
+How to create a new Expectation Suite without the CLI
+***************************************************
+
+In some environments, you might not be able to use the :ref:`CLI <command_line>` to create a new Expectation Suite. This guide shows how to create a new Expectation Suite and start adding Expectations using Python code, for example in a Jupyter notebook.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+    - Launched a generic notebook (e.g. ``jupyter notebook``, ``jupyter lab``, etc.)
+    - :ref:`Configured a Datasource <how_to_guides__configuring_datasources>`
```

Comment:
Strictly speaking, the DataSource isn't required.

We use it to fetch the Batch, but you could also get a validatable DataAsset using `from_pandas` or `ge.dataset.sparkdf_dataset.SparkDFDataset`.

We should cover those workflows somewhere. It's a judgement call whether to split those workflows into separate articles. This article is short enough that I lean towards keeping them together.

(In the future, we're going to strongly advise users to use DataConnectors to instantiate Batches, so that they'll always have the right metadata attached. But even then, it won't be strictly required---especially in workflows like this where the Suite is the only thing being stored.)

Reply:
I'm almost 1:1 sticking with the validation_playground notebook flow here, so I'd like to keep it as-is for consistency.